<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleLineSymbolWidget</class>
 <widget class="QWidget" name="SimpleLineSymbolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Line width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Line style</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineWidth"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="lineStyles"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ColorPickerButtont" name="colorBtn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Sample</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="sample">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButtont</class>
   <extends>QPushButton</extends>
   <header>ColorPickerButtont.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
